jwst/resample/resample_step.py

```diff
@@ -114,7 +114,30 @@ def process(self, input):
         return result
 
     @staticmethod
-    def _check_list_pars(vals, name, min_vals=None):
+    def check_list_pars(vals, name, min_vals=None):
```

Comment:
Does this function need to be public? Or maybe it should be moved to some `utils.py` module if used by several un-related modules?

Reply:
It's used by the ResampleSpecStep to check the same sort of input parameters the ResampleStep receives.  I made these functions public static methods so that the spec step could call them without actually inheriting from the imaging step.  They shouldn't be needed elsewhere.